pyposeidon/utils/pplot.py

```diff
@@ -368,7 +408,21 @@ def quiver(self, it=None, u=None, v=None, title=None, scale=0.1, color="k", **kw
             v = v.filled(fill_value=-99999)
             u = u.filled(fill_value=-99999)
 
-        for val in ["x", "y", "t", "it", "u", "v", "title", "tes", "xy", "scale", "mask", "color", "var"]:
+        for val in [
+            "x",
+            "y",
+            "t",
+            "it",
+            "u",
+            "v",
+            "title",
+            "tes",
+            "xy",
+            "scale",
+            "mask",
+            "color",
+            "var",
+        ]:
```

Comment:
This list appears 3 times. Maybe define it as a constant? And then also remove the `except: pass`es too?

Reply:
Yep this is ugly. I did it and forgot about it. Will do.